copy-this-template-directory/index.md

```diff
@@ -1,16 +1,924 @@
+ 
+ ***
+
+![GAS 05](/copy-this-template-directory/GAS_05.png "GAS 05")
+
+set type to web app. Description - anything. Execute as - Me. Who has access - anyone.
+Click `Deploy`.
+
+ ***
+
+![GAS 01](/copy-this-template-directory/GAS_01.png "GAS 01")
+
+ click `Authorize access`
+ ***
+
+![GAS 02](/copy-this-template-directory/GAS_02.png "GAS 02")
+
+ 
+Choose your account and then click "Allow".
+
+Some users (usually, users not within the organization) will see a scary-looking
+
+"Google hasn’t verified this app
+The app is requesting access to sensitive info in your Google Account. Until the developer (your email) verifies this app with Google, you shouldn't use it."
+
+Click "Advanced"
+
+Then click "Go to code (unsafe)"
+
+ 
+ ***
+
+![GAS 03](/copy-this-template-directory/GAS_03.png "GAS 03")
+
+click "Allow".
+
+If all is fine, you'll see next popup with URL of your published web app.
+
+Web app
+
+URL
+
+`https://script.google.com/macros/s/xxxxxxxxxxxxxxxxxxx/exec`
+
+Copy
+
+click `copy` to get this Web app URL (make sure that you are copying web app URL and not Library URL) and update Directus Environment Variable `GCALENDARHOOKURL` to this value
+
+
+ ***
+
+ Back to the script editor
+
+![GAS 06](/copy-this-template-directory/GAS_06.png "GAS 06")
+
+as shown by the arrow `1`, select function `listCalendars`. Click Run (as shown by the arrow `2`)
+
+you'll see the result of the function listCalendars run. Copy the ID of the calendar that you want to track. Usually, it's the same as an account email.
+
+&nbsp; 
+
+### Update Spreadsheet Config
+
+In your Spreadsheet go to the 'Config' sheet.
+
+set `calendar_id` to the id of your calendar.
+
+set `directus_url_proxy` to the URL of Flow "Google Calendar event Proxy"
+
+set `directus_url_webhook_from_g` to the URL of Flow "Process events from Google Calendar"
+
+set `pass` to the same value as in Directus Environment Variable `GCALENDARHOOKSECRET`
+
+&nbsp; 
+
+### Google Apps Script 
+
+Back to the script editor. We need to run manually couple of functions.
+
+select function `triggerResubscribeOnceWeek` (just like you selected listCalendars earlier). Click `Run`.
+
+If the run was successful, in the `config` sheet values next to `channel_id`, `resource_id` will be filled.
+
+select function `runManual_getSyncedEvents`. Click `Run`.
+
+If the run was successful, in the `config` sheet value next to `sync_token`, `resource_id` will be filled.
+
+
+> For most of the Google Workspace App, Apps Script has a specific library, like `CalendarApp` with easy-to-use functions. However, these functions don't have all the functionality available for API calls. Luckily it's possible to use Advanced Calendar Service - it's almost like calling API calls directly. Thanks to that we can subscribe to notifications and retrieve a list of new events using syncToken.
```

Comment:
No need for this to be in a quote. I also ask why we need this information to complete or understand this post. The sentence below does enough to explain what is needed.

Reply:
I thought it's not a quote, but rather a sidenote. This Information probably needs addition for complete understanding: "In the template script copy this service is already enabled. But if you want to create script from scratch, you need to enable it. 'Services' menu on the left panel, click + and choose Google Calendar."